mm2src/coins/eth.rs

```diff
@@ -524,6 +529,23 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    let mut tx = req.tx_hash;
+    if tx.len() < 2 {
+        return MmError::err(RawTransactionError::InvalidHashError(tx));
+    }
+    if tx.starts_with("0x") {
+        tx = (tx[2..]).to_string();
+    }
+    let hash = H256::from_str(&tx).map_to_mm(|e| RawTransactionError::InvalidHashError(e.to_string()))?;
+    let web3_tx = coin.web3.eth().transaction(TransactionId::Hash(hash)).compat().await?;
+    let web3_tx = web3_tx.or_mm_err(|| RawTransactionError::HashNotExist(tx))?;
```

Comment:
Also we should return the original `req.tx_hash` instead of probably changed `tx`.

Reply:
Thank you, it's done.